runtime/logging_test.go

```diff
@@ -141,6 +148,12 @@ func TestRequestLogging(t *testing.T) {
 		for _, ent := range entriesForReq(ents, i) {
 			if ent.Message == "Sent response." {
 				act := ent.Fields["resp_body"].(string)
+				if tc.acceptEncoding == "gzip" {
+					reader := bytes.NewReader([]byte(act))
+					gzreader, _ := gzip.NewReader(reader)
+					output, _ := io.ReadAll(gzreader)
+					act = string(output)
```

Comment:
This adjusts the test, but if you're running opa with
```
opa run --server --log-level debug
```
will it spill gzip binary data into your logs?

Reply:
yes if the client sends the request with `Accept-Encoding` header set to gzip, then it would do so. But isn't it the intended behaviour?

Reply:
😅 spewing binary data into the logs is not the intended behaviour. For the metrics endpoint, we'll log "binary payload" instead -- this is what the test was about. For the data endpoints, I think we should include the decompressed data in the debug logs.

Reply:
I suppose decompressing what we've previously compressed to log it is also kind of weird, and an indicator that logging should be wired differently. 🤔